src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -149,8 +149,10 @@ Netex field | NTFS file | NTFS field | Note
 --- | --- | --- | ---
 AccessibilityAssessment/@id | stops.txt | equipment_id | 
 AccessibilityAssessment/MobilityImpairedAccess | | | see (1) below
-AccessibilityAssessment/ limitations/AccessibilityLimitation/ WheelchairAccess | equipments.txt | wheelchair_boarding | see (2) below
-AccessibilityAssessment/ limitations/AccessibilityLimitation/ AudibleSignsAvailable | equipments.txt | audible_announcement | see (2) below
+AccessibilityAssessment/limitations/AccessibilityLimitation/WheelchairAccess | equipments.txt | wheelchair_boarding | see (2) below
+AccessibilityAssessment/limitations/AccessibilityLimitation/EscalatorFreeAccess | equipments.txt | escalator | see (2) below
+AccessibilityAssessment/limitations/AccessibilityLimitation/LiftFreeAccess | equipments.txt | elevator | see (2) below
```

Comment:
IMHO, those 2 propoerties should not be included in the spec, theses infos are not available

Reply:
Yes I saw that. I will modify the specs tomorrow (or they will be integrated into the various remaining developments).